.github/ISSUE_TEMPLATE/bw-unified.yml

```diff
@@ -0,0 +1,96 @@
+name: Bitwarden Unified Bug Report
+description: File a bug report
+labels: [bug, bitwarden-unified]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Please do not submit feature requests. The [Community Forums](https://community.bitwarden.com) has a section for submitting, voting for, and discussing product feature requests.
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps To Reproduce
+      description: How can we reproduce the behavior.
+      value: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. Click on '...'
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected Result
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: actual
+    attributes:
+      label: Actual Result
+      description: A clear and concise description of what is happening.
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots or Videos
+      description: If applicable, add screenshots and/or a short video to help explain your problem.
+  - type: textarea
+    id: additional-context
+    attributes:
+      label: Additional Context
+      description: Add any other context about the problem here.
+  - type: input
+    id: version
+    attributes:
+      label: Githash Version
+      description: Please go to https://{your-bitwarden-domain}/api/config and copy the gitHash version
+    validations:
+      required: true
+  - type: textarea
+    id: environment-details
+    attributes:
+      label: Environment Details
+      description: If Self-Hosted please provide some additional environment details.
+      placeholder: |
+        - Operating system: [e.g. Windows 10, Mac OS Catalina]
+        - Environment: [e.g. Docker, EKS, ECS, K8S]
+        - Hardware: [e.g. Intel 6-core, 8GB RAM]
+  - type: textarea
+    id: docker-compose-details
+    attributes:
+      label: Docker Compose Details
+      description: Please include the details from your server/docker-unified/docker-compose.yml
+      placeholder: |
+        # MariaDB Example
+          db:
+            environment:
+              MARIADB_USER: "bitwarden"
+              MARIADB_PASSWORD: *REDACTED*
+              MARIADB_DATABASE: "bitwarden_vault"
+              MARIADB_RANDOM_ROOT_PASSWORD: "true"
+          image: mariadb:10
+          restart: always
+          volumes:
+            - data:/var/lib/mysql
```

Comment:
unified can be run with a simple `docker run` command too, so might not want to expand this to be a bit more general.